### DISCUSSION TITLE
get_or_none function

### DISCUSSION THREAD

**Discussion by timedoutuk** [encode/orm#156] [Ideas]
I think it would be quite nice to have a `get_or_none()` function when querying objects. it just feels cleaner than having a try/except NoMatch clause.

I find it odd that this isn't already implemented so if there's a reason please let me know :)

**Comment by aminalaee** [comment]
Hey @EEKIM10 ,
The `.first()` method already does that: https://www.encode.io/orm/making_queries/#first

**Reply by timedoutuk** [reply]
Oh I see, thank you
